flows/engine/session.go

```diff
@@ -46,15 +48,17 @@ func NewSession(cache *AssetCache, assetURLs map[AssetItemType]string) flows.Ses
 	}
 }
 
-func (s *session) Assets() flows.SessionAssets              { return s.assets }
-func (s *session) Environment() utils.Environment           { return s.env }
-func (s *session) SetEnvironment(env utils.Environment)     { s.env = env }
-func (s *session) Contact() *flows.Contact                  { return s.contact }
-func (s *session) SetContact(contact *flows.Contact)        { s.contact = contact }
+func (s *session) Assets() flows.SessionAssets          { return s.assets }
+func (s *session) Environment() utils.Environment       { return s.env }
+func (s *session) SetEnvironment(env utils.Environment) { s.env = env }
+func (s *session) Trigger() flows.Trigger               { return s.trigger }
+func (s *session) Contact() *flows.Contact              { return s.contact }
+func (s *session) SetContact(contact *flows.Contact)    { s.contact = contact }
+
 func (s *session) FlowOnStack(flowUUID flows.FlowUUID) bool { return s.flowStack.hasFlow(flowUUID) }
 
-func (s *session) SetTrigger(flow flows.Flow, parentRun flows.FlowRun) {
-	s.flowTrigger = &flowTrigger{flow: flow, parentRun: parentRun}
+func (s *session) PushFlow(flow flows.Flow, parentRun flows.FlowRun) {
```

Comment:
Now that trigger means something more like triggers in rapidpro, need different name for this

Reply:
NextFlow (and `SetNextFlow`?)
PendingFlow?
BatterUpFlow?